Charter.txt

```diff
@@ -0,0 +1,39 @@
+<Group 3>
+
+Availability for meeting expectations:
+LL (typical weekly availability): 
+Saturday and Sunday 8am-8pm 
+Monday and Wednesday 4pm-8pm
+Tuesday and Thursday 5pm-6pm
+Friday 5pm-9pm
+
+Response time to communication expectations
+LL: 
+max response time 24 hours (email or discord) though preferably sooner
+
+Definition of done: when is a task complete? (hint: "I'm done with that but I didn't check it in to github" is not done.)
+LL: 
+accomplishes task, tested/checked, refactored and submitted to github (if code) 
+
+Scrum meeting format/frequency and distribution of documentation tasks (meeting minutes, updating task board)
+LL: 
+format - Discord?   frequency - three times a week? (the minimum he gave us)
+meeting minutes - make a shared document (Docs, Word, etc) in which to put the minutes, type them in as we go along
+Update taskboard as we go (I actually have a Google Jamboard set up that I thought we could use for this - or not, whatever works)

```

Comment:
I like jamboards. it might be more beneficial to use githubs's built in tasks in projects as it will directly link into branches, commits, and close out tasks on pr's

Reply:
That sounds good, as long as we can all get it figured out (it seems like none of us have a super strong grasp on github).  Tbh, the only reason I thought of Jamboards is the imagery of sticky notes.